src/Entity/RoutingProblem.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace AppBundle\Entity;
```

Comment:
Could you change the namespace, please? The `AppBundle\Entity` namespace is better suited for classes that are persisted in database. Maybe a `AppBundle\Vroom` package?